explore/src/main/scala/explore/targeteditor/AladinContainer.scala

```diff
@@ -37,26 +39,74 @@ final case class AladinContainer(
 }
 
 object AladinContainer {
-  type Props = AladinContainer
+  type Props       = AladinContainer
+  type World2PixFn = Coordinates => Option[(Double, Double)]
+  val DefaultWorld2PixFn: World2PixFn = (_: Coordinates) => None
 
   protected implicit val propsReuse: Reusability[Props] =
-    Reusability.by(x => (x.target, x.options, x.fov))
+    Reusability.by((x: Props) => (x.target, x.options, x.fov))
```

Comment:
Are there reasons for not including the other properties in the Reuseability?

Reply:
I guess I should add the callbacks, I just fear `Reuse` 😃